### PR TITLE
feat(loader): Implement new <Loader/> component

### PR DESCRIPTION
**Pull request by mastertheblaster**

### CODE REVIEW

**Comment by mastertheblaster** [comment]
@3LOK Please take a look.

File: stories/Loader.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                    <div><RadioGroup.Radio value="small">Small</RadioGroup.Radio></div>
+                    <div><RadioGroup.Radio value="medium">Medium</RadioGroup.Radio></div>
+                    <div><RadioGroup.Radio value="large">Large</RadioGroup.Radio></div>
+                </RadioGroup>
+                <p>Loader text:</p>
+                <div style={{width:'200px'}}>
+                    <Input value={this.state.text} onChange={e => this.setState({ text: e.target.value })}/>
+                </div>
+                <div style={{padding:'40px'}}>
+                    <Loader size={this.state.size} text={this.state.text}/>
```
Comment (3LOK) [inline stories/Loader.js]: I personally prefer the storybook to showcase all entire options, and not a radio button that allows selecting between them. It ensures users don't 'miss' features, and allow in the future to add tests using screenshots.

What do you think? @shahata what do you think?

Comment (mastertheblaster) [inline stories/Loader.js]: Ok, I got the point, the same like with button examples, correct ?

Comment (3LOK) [inline stories/Loader.js]: exactly.

**Comment by mastertheblaster** [comment]
Updated the story book to show all the usage cases.

**Pull request merged**